utils/components.py

```diff
@@ -471,6 +478,7 @@ def test_model_registry(
             token = token_file.readline()
 
         registry = ModelRegistry(
+            is_secure=False, # TODO: Helber - revert it
```

Comment:
Snyk will complain about this

Reply:
The Model Registry was broken. Matt recommended doing this just for the tests to pass. I'll revert it before the merge.

Reply:
@hbelmiro you can set this in your DSPA to have it be trusted in our shared cluster:

```yaml
spec:
  apiServer:
    cABundle:
      configMapKey: ca.crt
      configMapName: kube-root-ca.crt
```

Reply:
Alternativelly, you can add the contents of kube-root-ca.crt into the DSCI as it is explained [here](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.18/html/installing_and_uninstalling_openshift_ai_self-managed/working-with-certificates_certs#accessing-s3-compatible-object-storage-with-self-signed-certificates_certs)